src/web/rest/blocks.rs

```diff
@@ -0,0 +1,72 @@
+use std::sync::Arc;
+
+use actix_web::{
+    get,
+    http::header::ContentType,
+    web::{self, Data},
+    HttpResponse,
+};
+use serde::Deserialize;
+
+use crate::{
+    block::{precomputed::PrecomputedBlock, store::BlockStore},
+    store::IndexerStore,
+};
+
+#[derive(Deserialize)]
+struct Params {
+    limit: Option<u32>,
+}
+
+fn get_limit(limit: Option<u32>) -> u32 {
+    limit.map(|value| value.min(10)).unwrap_or(1)
```

Comment:
I know defaulting to 1 is at parity with current API, but should we default to something more reasonable?

Reply:
Agreed. This is a discussion that should happen. Resolving for now to merge